### PR TITLE
UI Changes TS-233

### PR DESCRIPTION
**Pull request by mohansgithub**

### CODE REVIEW

**Review by nakramesh — CHANGES_REQUESTED** [review]
change for UI to UI Ramesh

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 1]
```diff
@@ -14,8 +14,10 @@
 			<h1>Devops Training Nallaperumal TN	</h1>
 			<h1>Devops Training Nallaperumal TN	</h1>
 			<h1>DevOps Training Mohit Kambam</h1>
+			<h1>UI is developed by Ramesh	</h1>
+			<h1>DevOps Training Mohit Kambam</h1>
 			<h1> DevOps Training 12 09 2020 Mohit Kambam</h1>
-			My New Code is here
```
Comment (mydevopstrainings1) [inline src/main/webapp/WEB-INF/views/home.jsp:18]: Looks good to be I approve !

**Review by stnl22 — APPROVED** [review]
good to deploy

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 2]
```diff
@@ -14,8 +14,10 @@
 			<h1>Devops Training Nallaperumal TN	</h1>
 			<h1>Devops Training Nallaperumal TN	</h1>
 			<h1>DevOps Training Mohit Kambam</h1>
+			<h1>UI is developed by Ramesh	</h1>
```
Comment (mydevopstrainings1) [inline src/main/webapp/WEB-INF/views/home.jsp:17]: Remove TN

**Review by stnl22 — CHANGES_REQUESTED** [review]
please remove TN

**Comment by mohansgithub** [comment]
> change for UI to UI Ramesh

Added the change

File: src/main/webapp/WEB-INF/views/home.jsp (hunk 2 above)
Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp:17]: Ok Mohan

Comment (mydevopstrainings1) [inline src/main/webapp/WEB-INF/views/home.jsp:17]: Change UI to My UI

**Review by mydevopstrainings1 — COMMENTED** [review]
Mohan please fix this

File: src/main/webapp/WEB-INF/views/home.jsp (hunk 2 above)
Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp:17]: I fixed it

**Review by mydevopstrainings1 — CHANGES_REQUESTED** [review]
Updated the code

**Review by nakramesh — COMMENTED** [review]
Mohan add UI to UI_DEVOPS

**Review by theunforgivenme — CHANGES_REQUESTED** [review]
Pls make appropriate changes

**Review by theunforgivenme — APPROVED** [review]

**Review by mydevopstrainings1 — CHANGES_REQUESTED** [review]
Check and updated

**Review by nakramesh — COMMENTED** [review]
Again submitting

**Pull request merged**